doghouse/server/token.go

```diff
@@ -0,0 +1,17 @@
+package server
+
+import (
+	"crypto/rand"
+	"fmt"
+	"io"
+)
+
+func GenerateRepositoryToken() string {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function GenerateRepositoryToken should have comment or be unexported